app/views/submissions/browse.html.erb

```diff
@@ -76,3 +77,32 @@
 <% content_for :wrap_left_content do %>
   <div id='submissions_table'></div>
 <% end %>
+<script>
+  jQuery(document).ready(function() {
+    var downloadGroupingsFiles = function(){
+      window.location.href = "<%= download_groupings_files_assignment_submissions_path(
+          groupings: @groupings); %>";
+    };
+
+    jQuery('#download_all_submissions_link').on('click', function () {
+      jQuery.ajax({
+        url: '<%= check_collect_status_assignment_submissions_path(groupings: @groupings) %>',
+        type: 'GET',
+        success: function (data) {
+          if (data.collect_status !== true) {
+            var proceed = confirm('<%= t('collect_submissions.collection_incomplete_warning') %>');
+
+            if (proceed != true) {
```

Comment:
`Window.confirm()` always returns `true` if the user pressed OK and false otherwise -- the check could be simplified.